CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## 1.27.0
+
+* Adds an overloaded `map.set()` function:
```

Comment:
Can you add a link to the sass-lang.com documentation as well? It's not written yet, but when it is it'll be at https://sass-lang.com/documentation/modules/map#set.